src/sentry/static/sentry/app/views/releases/detail/organization/releaseOverview.jsx

```diff
@@ -67,14 +67,16 @@ export default class OrganizationReleaseOverview extends AsyncView {
 
     const deploys = this.state.deploys;
 
-    // TODO: Issue lists depend on organization issues endpoint
-    // const query = {};
+    // Unlike project releases) which filter issue lists on environment, we
+    // do not apply any of the global selection filters (project, environment,
+    // or time) to the organization release
```

Comment:
Why not?

Reply:
We decided that none of the filters (project/env/timestamp will be applied to this page, it would be odd for one of the filters to have an effect but not the other two

Reply:
So, it seems there will actually be a number of situations where some filters will apply but not others so this rationale doesn't really apply, nevertheless we agreed as a group this page won't be filtered (at least for v0) of this page